src/transport/SecureSessionMgr.cpp

```diff
@@ -138,7 +138,14 @@ CHIP_ERROR SecureSessionMgr::SendMessage(NodeId peerNodeId, System::PacketBuffer
 exit:
     if (msgBuf != NULL)
     {
-        ChipLogProgress(Inet, "Secure transport failed to encrypt msg %u: %s", state->GetSendMessageIndex(), ErrorStr(err));
+        if (state == nullptr)
```

Comment:
The compiler's hopefully smart enough to figure it out; however, I'd `const char * errstr = ErrorStr(err)` at the top of the common block and then use `errstr`, but that's a pedantic hand-optimization.

Reply:
Fixed. Feel free to be pedantic !